core/src/main/scala_spark_others/ai/h2o/sparkling/SparkTimeZone.scala

```diff
@@ -0,0 +1,25 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling
+
+import java.util.TimeZone
+import ai.h2o.sparkling.utils.SparkSessionUtils
+
+object SparkTimeZone {
+  def current(): TimeZone = TimeZone.getTimeZone(SparkSessionUtils.active.sessionState.conf.sessionLocalTimeZone)
```

Comment:
SparkTimeZone is same for Spark 2.2, 2.3 and spark_others. In this case we can just define spark_others

Reply:
Ok, will do.

Reply:
Ah, sorry, my bad. We need to have those files there. As long the specific directory for some spark exists, we need to have the classes there. Will create the followup fix